src/file-download/file-list-card/file-list-card.styles.tsx

```diff
@@ -0,0 +1,216 @@
+    width: 1.5rem;
+
+    ${(props) => {
+        if (props.$disabled) {
+            return css`
+                color: ${Color.Neutral[4]};
+            `;
+        }
+    }}
+`;
+
+export const Box = styled.div`
+    background: ${Color.Accent.Light[6]};
+    border: 1px solid ${Color.Neutral[5]};
+    border-radius: 4px;
+    padding: 1rem 2rem;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    cursor: pointer;
+    ${MediaQuery.MaxWidth.mobileL} {
+        padding: 1rem;
+    }
+    :hover {
+        background: ${Color.Accent.Light[5]};
+    }
+`;
+
+export const ContentSection = styled.div<ContentSectionStyleProps>`
+    display: flex;
+    flex: 1;
+    align-items: center;
+
+    ${MediaQuery.MaxWidth.mobileL} {
+        flex-direction: column;
+        width: 100%;
+        align-items: flex-start;
+    }
+
+    ${(props) => {
+        if (props.$hasThumbnail) {
+            return css`
+                ${MediaQuery.MaxWidth.mobileL} {
+                    flex-direction: row;
+                    align-items: center;
+                }
+            `;
+        }
+    }}
+`;
+
+export const NameSection = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    width: 100%;
+    max-width: 22.688rem;
+`;
+
+export const ExtendedNameSection = styled.div`
+    display: flex;
+    flex: 1;
+    align-items: center;
+
+    ${MediaQuery.MaxWidth.mobileL} {
+        flex-direction: column;
+        align-items: flex-start;
+        width: 100%;
+    }
+`;
+
+export const FileSizeSection = styled.div<FileSizeSectionStyleProps>`
+    display: flex;
+    width: 5rem;
+    margin-left: 0.5rem;
+    justify-content: flex-end;
+
+    ${MediaQuery.MaxWidth.mobileL} {
+        ${(props) => {
+            if (props.$hideInMobile) {
+                return css`
+                    display: none;
+                    visibility: hidden;
+                `;
+            } else {
+                return css`
+                    width: 100%;
+                    margin-left: 0;
+                    margin-top: 0.5rem;
+                    justify-content: flex-start;
+                `;
+            }
+        }}
+    }
+`;
+
+export const ItemText = styled(Text.BodySmall)``;
+export const ItemDescriptionText = styled(ItemText)`
+    margin-top: 0.25rem;
+`;
+
+export const BaseErrorMessage = styled(Text.XSmall)`
+    font-size: 0.875rem !important;
+    color: ${Color.Validation.Red.Text};
+`;
+
+export const DesktopErrorMessage = styled(BaseErrorMessage)`
+    margin-top: 0.25rem;
+    ${MediaQuery.MaxWidth.mobileL} {
+        display: none;
+        visibility: hidden;
+    }
+`;
+
+export const MobileErrorMessage = styled(BaseErrorMessage)`
+    display: none;
+    visibility: hidden;
+    ${MediaQuery.MaxWidth.mobileL} {
+        display: block;
+        visibility: visible;
+        margin-top: 0.5rem;
+    }
+`;
+
+export const Spinner = styled(ComponentLoadingSpinner)<MainStyleProps>`
```

Comment:
isn't this a duplicate from another place? why not use the component directly?

Reply:
it has some style that is fixed in that component might not fit in this one

Reply:
can you elaborate? because from what I can see, `Spinner` is manually overriding the color. why not pass color prop directly to `ComponentLoadingSpinner`?

there's also no need for the buttonStyle as only a single fixed color is needed here